test/parallel/test-https-host-headers.js

```diff
@@ -55,7 +55,7 @@ function testHttps() {
       method: 'GET',
       path: `/${counter++}`,
       host: 'localhost',
-      //agent: false,
+      // agent: false,
```

Comment:
Suggestion: remove this and the other cases as well. There is no comment why this is kept in here.